lib/usd/ui/TreeModel.cpp

```diff
@@ -15,386 +15,360 @@
 //
 #include "TreeModel.h"
 
-#include <functional>
-
-#include <QtWidgets/QTreeView>
-#include <QtCore/QSortFilterProxyModel>
-
 #include <pxr/usd/usd/variantSets.h>
 
-#include <mayaUsdUI/ui/TreeItem.h>
-#include <mayaUsdUI/ui/ItemDelegate.h>
+#include <QtCore/QSortFilterProxyModel>
+#include <QtWidgets/QTreeView>
```

Comment:
It seems like these should be part of group 6 and thus be moved to line 25?

Reply:
Good catch. This is because we don't recognize mayaUsdUI in https://github.com/Autodesk/maya-usd/blob/dev/.clang-format#L56. I will fix this and create a new PR.

Reply:
PR opened for it https://github.com/Autodesk/maya-usd/pull/888